### DISCUSSION TITLE
Is there any way to see the certificates managed by the function app?

### DISCUSSION THREAD

**Discussion by daraii** [shibayan/appservice-acmebot#407] [Q&A] [answered]
Is there something similar to the dashboard for the KeyVault func app? I'd like to list the certificates being managed by the acmebot.

**Comment by shibayan** [comment, marked as answer]
There are currently no plans to implement a dashboard of any kind, but certificates under Acmebot's control can be queried using the Resource Graph.

```
resources
| where type == "microsoft.web/certificates" and tags["Issuer"] == "Acmebot"
```
